packages/libs/react-ui/src/components/Footer/Footer.stories.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  IFooterIconItemProps,
+  IFooterLinkItemProps,
+  SystemIcon,
+} from './../../';
+import { colorVariants } from './Footer.css';
+import { Footer, IFooterProps } from './index';
+
+import { Target } from '@components/Footer/FooterLinkItem';
+import { IconType } from '@components/Icon/IconWrapper';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+const meta: Meta<
+  {
+    linksCount: number;
+    iconsCount: number;
+  } & IFooterProps &
+    IFooterLinkItemProps &
+    IFooterIconItemProps
+> = {
+  title: 'Layout/Footer',
```

Comment:
I think this would be better grouped in a new section like `Navigation/Footer` because layout components are mainly for positioning